src/polymer-project.ts

```diff
@@ -62,20 +62,17 @@ export interface ProjectOptions {
   sourceGlobs?: string[];
 
   /**
-   * List of glob patterns, relative to root, of dependencies to read from the
-   * file system. For example node_modules\/**\/* and bower_components\/**\/*
+   * List of file paths, relative to the project directory, that should be included
+   * as dependencies in the build target.
    */
-  dependencyGlobs?: string[];
+  includeDependencies?: string[];
 }
 
 export const defaultSourceGlobs = [
-  '**/*',
-  '!build/**/*',
-];
-
-export const defaultDependencyGlobs = [
-  'bower_components/**/*',
-  'node_modules/**/*',
+  'src/**/*',
+  // NOTE(fks) 06-29-2016: `polymer-cli serve` uses a bower.json file to display
+  // information about the project. The file is included here by default.
```

Comment:
I know I agree it seems unnecessary. It would be nice if it was optional, and if the bower.json file wasn't present it would just not display that extra component-style information. 

I'll create a quick PR on polymer-serve to make it optional, but lets make that change in a future PR / not have that block this.